### ISSUE TITLE
ModuleNotFoundError: No module named 'second.core.non_max_suppression.nms'

### ISSUE THREAD

**Issue by snj00070** [traveller59/second.pytorch#223]
nvcc -std=c++11 -c -o ../cc/nms/nms_kernel.cu.o ../cc/nms/nms_kernel.cu.cc -I/usr/local/cuda/include -x cu -Xcompiler -fPIC -arch=sm_53 --expt-relaxed-constexpr 
In file included from /usr/local/cuda/include/cuda_runtime.h:115:0,
                 from <command-line>:0:
/usr/local/cuda/include/crt/common_functions.h:74:0: warning: "__CUDACC_VER__" redefined
 #define __CUDACC_VER__ "__CUDACC_VER__ is no longer supported.  Use __CUDACC_VER_MAJOR__, __CUDACC_VER_MINOR__, and __CUDACC
 ^
<command-line>:0:0: note: this is the location of the previous definition
/usr/lib/gcc/x86_64-linux-gnu/5/include/mwaitxintrin.h(36): error: identifier "__builtin_ia32_monitorx" is undefined

/usr/lib/gcc/x86_64-linux-gnu/5/include/mwaitxintrin.h(42): error: identifier "__builtin_ia32_mwaitx" is undefined

/usr/include/c++/5/bits/stl_iterator_base_types.h(154): error: name followed by "::" must be a class or namespace name
          detected during:
            instantiation of class "std::__iterator_traits<_Iterator, void> [with _Iterator=int]" 
(163): here
            instantiation of class "std::iterator_traits<_Iterator> [with _Iterator=int]" 
/usr/include/boost/token_functions.hpp(359): here

/usr/include/c++/5/bits/stl_iterator_base_types.h(155): error: name followed by "::" must be a class or namespace name
          detected during:
            instantiation of class "std::__iterator_traits<_Iterator, void> [with _Iterator=int]" 
(163): here
            instantiation of class "std::iterator_traits<_Iterator> [with _Iterator=int]" 
/usr/include/boost/token_functions.hpp(359): here

/usr/include/c++/5/bits/stl_iterator_base_types.h(156): error: name followed by "::" must be a class or namespace name
          detected during:
            instantiation of class "std::__iterator_traits<_Iterator, void> [with _Iterator=int]" 
(163): here
            instantiation of class "std::iterator_traits<_Iterator> [with _Iterator=int]" 
/usr/include/boost/token_functions.hpp(359): here

/usr/include/c++/5/bits/stl_iterator_base_types.h(157): error: name followed by "::" must be a class or namespace name
          detected during:
            instantiation of class "std::__iterator_traits<_Iterator, void> [with _Iterator=int]" 
(163): here
            instantiation of class "std::iterator_traits<_Iterator> [with _Iterator=int]" 
/usr/include/boost/token_functions.hpp(359): here

/usr/include/c++/5/bits/stl_iterator_base_types.h(158): error: name followed by "::" must be a class or namespace name
          detected during:
            instantiation of class "std::__iterator_traits<_Iterator, void> [with _Iterator=int]" 
(163): here
            instantiation of class "std::iterator_traits<_Iterator> [with _Iterator=int]" 
/usr/include/boost/token_functions.hpp(359): here

Segmentation fault (core dumped)
concurrent.futures.process._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/home/harry/second.pytorch/second/core/non_max_suppression/nms_cpu.py", line 10, in <module>
    from second.core.non_max_suppression.nms import (
ModuleNotFoundError: No module named 'second.core.non_max_suppression.nms'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/harry/anaconda3/lib/python3.6/concurrent/futures/process.py", line 175, in _process_worker
    r = call_item.fn(*call_item.args, **call_item.kwargs)
  File "/home/harry/anaconda3/lib/python3.6/concurrent/futures/process.py", line 153, in _process_chunk
    return [fn(*args) for args in chunk]
  File "/home/harry/anaconda3/lib/python3.6/concurrent/futures/process.py", line 153, in <listcomp>
    return [fn(*args) for args in chunk]
  File "/home/harry/second.pytorch/second/utils/buildtools/command.py", line 255, in compile_func
    raise RuntimeError("compile failed with retcode", ret.returncode)
RuntimeError: ('compile failed with retcode', 139)
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "create_data.py", line 9, in <module>
    from second.core import box_np_ops
  File "/home/harry/second.pytorch/second/core/box_np_ops.py", line 7, in <module>
    from second.core.non_max_suppression.nms_gpu import rotate_iou_gpu_eval
  File "/home/harry/second.pytorch/second/core/non_max_suppression/__init__.py", line 1, in <module>
    from second.core.non_max_suppression.nms_cpu import nms_jit, soft_nms_jit
  File "/home/harry/second.pytorch/second/core/non_max_suppression/nms_cpu.py", line 18, in <module>
    cuda=True)
  File "/home/harry/second.pytorch/second/utils/buildtools/pybind11_build.py", line 113, in load_pb11
    cmds, cwd, num_workers=num_workers, compiler=compiler)
  File "/home/harry/second.pytorch/second/utils/buildtools/command.py", line 277, in compile_libraries
    if any([r.returncode != 0 for r in rets]):
  File "/home/harry/second.pytorch/second/utils/buildtools/command.py", line 277, in <listcomp>
    if any([r.returncode != 0 for r in rets]):
  File "/home/harry/anaconda3/lib/python3.6/concurrent/futures/process.py", line 366, in _chain_from_iterable_of_lists
    for element in iterable:
  File "/home/harry/anaconda3/lib/python3.6/concurrent/futures/_base.py", line 586, in result_iterator
    yield fs.pop().result()
  File "/home/harry/anaconda3/lib/python3.6/concurrent/futures/_base.py", line 425, in result
    return self.__get_result()
  File "/home/harry/anaconda3/lib/python3.6/concurrent/futures/_base.py", line 384, in __get_result
    raise self._exception
RuntimeError: ('compile failed with retcode', 139)

How should I go about this issue? please help! thanks

**Comment by snj00070** [comment]
fix by reinstalling cuda version 10.0 nvidia driver version 410.48 on rtx2080ti. 
reinstallation of ubuntu is recomended

**Comment by sarimmehdi** [comment]
Hello. Can you tell me the solution to this? I was able to follow the instructions and everything installed just fine. Then, I ran `python create_data.py kitti_data_prep --data_path=KITTI_DATASET_ROOT` but I get the error:

```
Traceback (most recent call last):
  File "/home/sarim/PycharmProjects/DETS3D/second.pytorch/second/core/non_max_suppression/nms_cpu.py", line 10, in <module>
    from second.core.non_max_suppression.nms import (
ModuleNotFoundError: No module named 'second.core.non_max_suppression.nms'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/sarim/PycharmProjects/DETS3D/second.pytorch/second/create_data.py", line 9, in <module>
    from second.core import box_np_ops
  File "/home/sarim/PycharmProjects/DETS3D/second.pytorch/second/core/box_np_ops.py", line 7, in <module>
    from second.core.non_max_suppression.nms_gpu import rotate_iou_gpu_eval
  File "/home/sarim/PycharmProjects/DETS3D/second.pytorch/second/core/non_max_suppression/__init__.py", line 1, in <module>
    from second.core.non_max_suppression.nms_cpu import nms_jit, soft_nms_jit
  File "/home/sarim/PycharmProjects/DETS3D/second.pytorch/second/core/non_max_suppression/nms_cpu.py", line 18, in <module>
    cuda=True)
  File "/home/sarim/PycharmProjects/DETS3D/second.pytorch/second/utils/buildtools/pybind11_build.py", line 90, in load_pb11
    arch = find_cuda_device_arch()
  File "/home/sarim/PycharmProjects/DETS3D/second.pytorch/second/utils/find.py", line 77, in find_cuda_device_arch
    cuda_home = find_cuda()
  File "/home/sarim/PycharmProjects/DETS3D/second.pytorch/second/utils/find.py", line 69, in find_cuda
    "No CUDA runtime is found, using CUDA_HOME='{}'".format(cuda_home))
RuntimeError: No CUDA runtime is found, using CUDA_HOME='None'
```
It seems to me that there is no file called nms in second/core/non_max_suppression
What would you recommend as a solution to this?
